### PR TITLE
KOGITO-5306: implement process monitoring operational dashboard

### PR DESCRIPTION
**Pull request by kostola**
- [JIRA task](https://issues.redhat.com/browse/KOGITO-5306)

### CODE REVIEW

**Comment by danielezonca** [comment]
@cristianonicolai @tiagodolphine @krisv 
FYI
This is a very initial impl that extends process metrics and produce a basic grafana dashboard. It covers the end to end only for strings

**Comment by danielezonca** [comment]
@kostola 
Can you please have a look to the failure of `Example build`? It looks related, I was expecting the PR to work without the need to change downstream repo 🤔

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by danielezonca — APPROVED** [review]

File: kogito-codegen-modules/kogito-codegen-processes/src/main/resources/grafana-dashboard-template/processes/global-operational-dashboard-template.json [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+          "values": false
+        },
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto"
+      },
+      "pluginVersion": "6.6.1",
+      "targets": [
+        {
+          "expr": "sum(kie_process_instance_started_total)",
```
Comment (cristianonicolai) [inline kogito-codegen-modules/kogito-codegen-processes/src/main/resources/grafana-dashboard-template/processes/global-operational-dashboard-template.json:59]: @kostola can you review the naming as I changed the prefix recently
```suggestion
          "expr": "sum(kogito_process_instance_started_total)",
```

Comment (kostola) [reply kogito-codegen-modules/kogito-codegen-processes/src/main/resources/grafana-dashboard-template/processes/global-operational-dashboard-template.json]: Fixed :+1: (I didn't realize you already merged the changes to the metrics names)

**Comment by cristianonicolai** [comment]
Jenkins rerun runtimes tests

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**